.github/workflows/build_wheels.yml

```diff
@@ -79,7 +76,7 @@ jobs:
           python-version: '3.9'
 
       - name: Install build requirements
-        run: python3 -m pip --disable-pip-version-check install scikit-build cmake -r requirements.txt --upgrade --progress-bar=off
+        run: python3 -m pip --disable-pip-version-check install scikit-build cmake -r .github/workflows/conda-envs/requirements-test.txt --upgrade --progress-bar=off
```

Comment:
The requirements for _building_ should be minimal. The full set of testing requirements should not be installed at build time.

Reply:
Would you rather I install just the build requirements to build, and then before testing do another `pip install` for the test-specific requirements? We need the testing requirements to be installed at some point.

Reply:
Yes, that's exactly what I would recommend.